sds011_reader:
  data_logging_folder: "/home/robert/FER/EKO-KOMVOZ/SDS011_senzor/SDS011_data/ROS_driver_tests"
  front_sensor_id: 24720
  back_sensor_id: 55687
  front_sensor_serial_file: "/dev/ttyUSB0"
  back_sensor_serial_file: "/dev/ttyUSB3"

  front_sensor_relay_serial_file: "/dev/ttyUSB1"
  back_sensor_relay_serial_file: "/dev/ttyUSB4"
